### PR TITLE
#AV-97946 travis Python version updated to Python3.7

### PR DESCRIPTION
**Pull request by shounak-opcito**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
       install:
         - export AVI_CONTROLLER="10.79.169.56"
         - export AVI_USERNAME="admin"
         - export AVI_PASSWORD="password"
         - export API_VERSION="18.2.8"
         - export PYTHONPATH=$PWD
         - export ANSIBLE_LIBRARY=$PWD/
-        - pip install ansible==2.8.5
-        - pip install vcrpy pytest requests avisdk
+        - pip install ansible==2.9.0
```
Comment (chaitanyaavi) [inline .travis.yml:14]: use 2.9.14

Comment (shounak-opcito) [reply .travis.yml:14]: @chaitanyaavi as per the discussion we are keeping this as 2.9.0 or should I upgrade it here?

Comment (chaitanyaavi) [reply .travis.yml:14]: We are keeping it 2.9.0 in controller here we need to keep it latest on 2.9

Comment (shounak-opcito) [reply .travis.yml]: updated ansible version to 2.9.14

**Review by chaitanyaavi — APPROVED** [review]

**Review by sabandi — APPROVED** [review]

**Pull request merged**